compiler/src/test/java/com/squareup/anvil/compiler/MergeModulesTest.kt

```diff
@@ -148,7 +148,7 @@ class MergeModulesTest {
       import com.squareup.anvil.annotations.compat.MergeModules
 
       @ContributesTo(Any::class)
-      abstract class DaggerModule1
+      abstract class DaggerModule2
```

Comment:
This was wrong and I don't know why it started to blow up now. Notice how the same class name was used below.